src/main/java/bio/terra/cli/command/workspace/Create.java

```diff
@@ -16,7 +17,9 @@ public class Create extends BaseCommand {
   // Variable is `id` instead of `userFacingId` because user sees it with `terra workspace create`
   private String id;
 
-  @CommandLine.Mixin WorkspaceNameAndDescription workspaceNameAndDescription;
+  @CommandLine.Mixin WorkspaceNameAndDescription workspaceNameAndDescriptionAndProperty;
```

Comment:
workspaceNameAndDescriptionAndProperty -> workspaceNameAndDescription

Tip: Before asking reviewers to take another look, you may want look at your changes, to see if anything needs updating :)

Reply:
sorry for this one, I use refactor to change the name, not sure why it still here and no fail during the check...